syllabus/mermaid.md

```diff
@@ -0,0 +1,46 @@
+# Mermaid Overview
+
+## What is Mermaid?
+
+Mermaid is a flowchart and diagram visualization tool based on JavaScript and uses Markdown syntax to create flowcharts that are extremely dynamic and high quality. Mermaid is especially useful because you are able to display your beautiful flowcharts using backticks to create a code block. 
```

Comment:
can you link to the docs?

Reply:
Do you mean linking to repo? https://github.com/mermaid-js/mermaid/tree/develop/docs or our docs?

Reply:
the rendered website of the mermaid documentation